bot/exts/utils/source.py

```diff
@@ -0,0 +1,133 @@
+import inspect
+from pathlib import Path
+from typing import Optional, Tuple, Union
+
+from discord import Embed, utils
+from discord.ext import commands
+
+from bot.bot import ThreadBot
+from bot.constants import URLs
+
+SourceType = Union[commands.HelpCommand, commands.Command, commands.Cog, str, commands.ExtensionNotLoaded]
```

Comment:
Isn't `commands.ExtensionNotLoaded` an error?

Reply:
This was some code I missed when removing unneded things when copying from the bot. ext not loaded is reaised in that command then you try to get the source of a tag and the tags cog isn't loaded.

3003ac9